.travis.yml

```diff
@@ -1,9 +1,7 @@
 rvm:
 - 2.2.0
-- 2.3.0
-- 2.4.2
-- 2.5.0
```

Comment:
I would keeps supporting all Ruby versions that didn't reach their EOL.

Reply:
What I wanted to do is to have sufficient "spread" of versions but run less builds. So I've set the lowest supported and the newest supported, assuming that versions in between will kinda work 🕵🏻‍♂️ Do you think it is safe enough, or would you prefer us test with all minor Ruby versions?

Reply:
I would do with all latest minor versions TBH:

```
- 2.4.6
- 2.5.5
- 2.6.3
- jruby
```

Reply:
Okay, let's do that until we bump major